install_config/monitoring/configuring-openshift-cluster-monitoring.adoc

```diff
@@ -34,6 +34,12 @@ The {product-title} Ansible `openshift_cluster_monitoring_operator` role configu
 |`openshift_cluster_monitoring_operator_alertmanager_storage_enabled`
 | Enable persistent storage of Alertmanager notifications and silences. This variable is set to `false` by default.
 
+|`openshift_cluster_monitoring_operator_prometheus_storage_class_name`
+| Set to the desired a specific xref:../persistent_storage/dynamically_provisioning_pvs.html#defining-storage-classes[StorageClass] to ensure that pods are configured the `PVC` with specific `storageclass`. Defaults to `none`.
```

Comment:
need some change for the description of openshift_cluster_monitoring_operator_prometheus_storage_class_name and openshift_cluster_monitoring_operator_alertmanager_storage_class_name 

default value is none, then it will use the default storage class name

So the description should be:
Set to the desired a specific StorageClass to ensure that pods are configured the PVC with specific storageclass. Defaults to none, it will use the default storage class name.

Reply:
Thank you for your review and suggestion @juzhao, I updated along your suggestion.